<?php

use Symfony\Component\DependencyInjection\Argument\RewindableGenerator;
use Symfony\Component\DependencyInjection\Exception\RuntimeException;

// This file has been auto-generated by the Symfony Dependency Injection Component for internal use.
// Returns the private '.service_locator.Oy.nOIY' shared service.

return $this->privates['.service_locator.Oy.nOIY'] = new \Symfony\Component\DependencyInjection\Argument\ServiceLocator($this->getService, [
    'newparte' => ['privates', '.errored..service_locator.Oy.nOIY.App\\Entity\\Utilisateur', NULL, 'Cannot autowire service ".service_locator.Oy.nOIY": it references class "App\\Entity\\Utilisateur" but no such service exists.'],
    'parte' => ['privates', '.errored..service_locator.Oy.nOIY.App\\Entity\\Utilisateur', NULL, 'Cannot autowire service ".service_locator.Oy.nOIY": it references class "App\\Entity\\Utilisateur" but no such service exists.'],
    'violations' => ['privates', '.errored..service_locator.Oy.nOIY.Symfony\\Component\\Validator\\ConstraintViolationList', NULL, 'Cannot autowire service ".service_locator.Oy.nOIY": it references class "Symfony\\Component\\Validator\\ConstraintViolationList" but no such service exists.'],
], [
    'newparte' => 'App\\Entity\\Utilisateur',
    'parte' => 'App\\Entity\\Utilisateur',
    'violations' => 'Symfony\\Component\\Validator\\ConstraintViolationList',
]);
